spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/logging/StackdriverJsonLayoutLoggerTests.java

```diff
@@ -30,6 +30,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
+ * Tests the Stackdriver Json Layout Logger.
```

Comment:
I think this should be fine; it makes sense and isn't flagrantly wrong. It will be a scary day when robotic code style starts creeping into human language.